src/main/java/pro/sky/exever/examiner/JavaQuestionService.java

```diff
@@ -0,0 +1,55 @@
+package pro.sky.exever.examiner;
+
+import org.springframework.stereotype.Service;
+import pro.sky.exever.examiner.exception.QuestionsIsEmptyException;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+
+@Service
+public class JavaQuestionService implements QuestionService {
+    private final JavaQuestionRepository javaQuestionRepository;
+
+    public JavaQuestionService(JavaQuestionRepository javaQuestionRepository) {
```

Comment:
```suggestion
    public JavaQuestionService(QuestionRepository javaQuestionRepository) {
```
инъекцию надо делать по абстракции всегда (принцип L в SOLID)